Makefile

```diff
@@ -398,6 +398,4 @@ project/validate:
 	[ "$$(ls -A $(PWD))" ] || (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
 	@ echo "[$@]: Target test folder validation successful"
 
-install: black/install pylint/install pylint-pytest/install pydocstyle/install yamllint/install cfn-lint/install bumpversion/install
```

Comment:
We can add this back now, including the targets we removed earlier for the Docker-based tools.

Reply:
@lorengordon made the changes but I'm not sure if the naming conventions are correct..